egs/babel_multilang/s5/local/nnet3/run_shared_ivector_extractor.sh

```diff
@@ -12,8 +12,8 @@ stage=4
 suffix=_sp
 feat_suffix=_hires # feat_suffix used in train_set for lda_mllt training.
 nnet3_affix=
-numLeavesMLLT=5500
-numGaussMLLT=90000
+numLeavesMLLT=2500
```

Comment:
These parameters are defined in lang.conf (for babel languages) and I set the initial values to be equal to values in lang.conf.